katas/es6/language/generators/yield.js

```diff
@@ -0,0 +1,38 @@
+// Generators - Yield Statements
+// To do: make all tests pass, leave the assert lines unchanged!
+
+describe('generator instance', () => {
+  
+  function* generator() {
+    yield 'hello';
+    yield 'world';
+  }
+  
+  // create instance
+  let genInstance = generator();
+  
+  it('should have an undefined value property after initialisation', () => {
+    let value = genInstance;
```

Comment:
I don't think that this test is really leading towards gaining knowledge. It was irritating to me. The first test should be simple, to understand and fix. So the person gets hooked. 
Besides that the `genInstance.value` is getting the `value` property from the iterator, not from a result of a `next()` call on the iterator.